easybuild/easyblocks/r/r.py

```diff
@@ -120,6 +120,23 @@ def make_module_req_guess(self):
 
         return guesses
 
+    def make_module_extra(self):
+        """Add extra environment variables to modulefile"""
+
+        txt = super(EB_R, self).make_module_extra()
+
+        pythonroot = get_software_root('Python')
+        if pythonroot:
+            # make sure EB-provided Python is used, and that reticulate does not install it's own Python
+            # see: https://rstudio.github.io/reticulate/reference/use_python.html
+            # see: https://github.com/rstudio/reticulate/issues/894
+            ext_reticulate = [ext[0] for ext in self.cfg['exts_list'] if ext[0] == 'reticulate']
+            if ext_reticulate:
+                txt += self.module_generator.set_environment(
+                    'RETICULATE_PYTHON', os.path.join(pythonroot, 'bin', 'python'))
```

Comment:
Hmm, wouldn't it make more sense to have a custom easyblock for `reticulate` for this?

As it is done now, this will only work when installing `reticulate` as an extension for `R`?

Reply:
yes, good point.